src/multidim/intervalbox.jl

```diff
@@ -1,12 +1,22 @@
 # This file is part of the IntervalArithmetic.jl package; MIT licensed
 
-doc"""An `IntervalBox` is an $N$-dimensional rectangular box, given
-by a Cartesian product of $N$ `Interval`s.
-"""
 
 
-immutable IntervalBox{N,T} <: StaticVector{Interval{T}}
-    data::NTuple{N,Interval{T}}
+if VERSION >= v"0.6.0-dev"
+
+    doc"""An `IntervalBox` is an $N$-dimensional rectangular box, given
+    by a Cartesian product of $N$ `Interval`s.
+    """
+    immutable IntervalBox{N,T} <: StaticVector{N, Interval{T}}
+        data::NTuple{N,Interval{T}}
+    end
+else
+    doc"""An `IntervalBox` is an $N$-dimensional rectangular box, given
+    by a Cartesian product of $N$ `Interval`s.
+    """
+    immutable IntervalBox{N,T} <: StaticVector{Interval{T}}
+        data::NTuple{N,Interval{T}}
+    end
 end
```

Comment:
There is (currently)  no way to specify in REQUIRE that  different versions of StaticArrays.jl should be used on 0.5 and 0.6 without making separate branches. 

We can specify in the docs which version is required and impose the correct minimum version once we drop support for 0.5.

What problem are you having locally?